include/jemalloc/internal/witness.h

```diff
@@ -29,6 +29,7 @@
 #define WITNESS_RANK_PROF_TDATA		8U
 #define WITNESS_RANK_PROF_GCTX		9U
 
+#define WITNESS_RANK_EXTENT_GROW_NEXT	10U
```

Comment:
I think make it different could be helpful, especially given that background thread triggers all sorts of reentrancy behaviors.